es-svc.yaml

```diff
@@ -10,7 +10,9 @@ spec:
   selector:
     component: elasticsearch
     role: client
+  clusterIP: 10.0.0.110
   ports:
   - name: http
+    nodePort: 30101
```

Comment:
Well you can hit the API server or use `kubectl proxy` command. Having this enforced to every user is not something I'd like to have around.